src/ethereum_test_tools/spec/blockchain/types.py

```diff
@@ -1171,3 +1171,17 @@ def format(cls) -> FixtureFormats:
         fixture.
         """
         return FixtureFormats.BLOCKCHAIN_TEST_HIVE
+
+
+def contains_exception(
```

Comment:
I think we could instead implement `def __contains__(self, item) -> bool` in `class ExceptionBase` (see [docs.python.org](https://docs.python.org/2/reference/datamodel.html#object.__contains__)), which could simply compare for equality, and we should be able to just check by doing `BlockException.RLP_ENCODING in block.exception`.

Reply:
I referenced a super old python version in here :see_no_evil: 
Correct link: https://docs.python.org/3.10/reference/datamodel.html#object.__contains__